src/project-manager.ts

```diff
@@ -382,9 +388,10 @@ class InMemoryLanguageServiceHost implements ts.LanguageServiceHost {
         this.versions = versions;
         this.projectVersion = 1;
         this.files = [];
-        // adding content of default library file read from the local file system 
-        const defaultLib = util.normalizePath(this.getDefaultLibFileName(options));
-        this.fs.entries[defaultLib] = fs_.readFileSync(defaultLib).toString();
+        // adding library files from the local file system
+        readTsLibraries().forEach((content, name) => {
```

Comment:
Actually, it's quite strange to me that this works. The TS library paths are all absolute on my local machine, so this ends up adding a bunch of files to the in-memory FS that begin with `/home/beyang/...`. Somehow, the TypeScript compiler fetches these file paths from the local FS, even though it should be agnostic about the local FS when talking to the in-memory language host.